base/namedtuple.jl

```diff
@@ -368,7 +368,24 @@ julia> Base.setindex(nt, "a", :a)
 function setindex(nt::NamedTuple, v, idx::Symbol)
     merge(nt, (; idx => v))
 end
+"""
```

Comment:
Would be good to add line breaks around the new definition.

Reply:
Might be worth folding this docstring into the one for `setindex(nt::NamedTuple, val, key::Symbol)`?